### DISCUSSION TITLE
Does it matters the machine used to create the initrd/initramfs file?

### DISCUSSION THREAD

**Discussion by Masber** [dracutdevs/dracut#1893] [General]
Dear dracut community,

I have a diskless cray server with some special hardware which I would like to boot with Rocky Linux. I am thinking about booting a Rocky Linux VM on my laptop to create the initrd/nitramfs file with the modules need (NFS and networking). The issue is I am not sure whether the kernel will be able to use the nics to find the rootfs in the NFS server. 

Does it matter where the initrd/initramfs file is created? Should I be able to use the initrd/initramfs created in my laptop and use it in a cray server?

thank  you

**Comment by LaszloGombos** [comment]
> Should I be able to use the initrd/initramfs created in my laptop and use it in a cray server?

Yes, this is possible (in a technical sense), but you need to be very mindful about the command line arguments passed to dracut to create initramfs. Your laptop needs to have the same version of kernel modules installed as your cray server.
